exe/kubernetes-restart

```diff
@@ -23,7 +23,7 @@ namespace = ARGV[0]
 context = ARGV[1]
 
 restart = KubernetesDeploy::RestartTask.new(namespace: namespace, context: context,
-   max_watch_seconds: max_watch_seconds)
+   global_timeout: global_timeout)
```

Comment:
Doesn't this need to be `global_timeout: max_watch_seconds`?

And do we need to update this line as well? If yes, it's also still present in the `deploy` and `run` exes. https://github.com/Shopify/kubernetes-deploy/blob/6c6f930cecf3fda21b6fb57cca4f7ab39455d0c9/exe/kubernetes-restart#L15

Reply:
We don't need to update the flag name here, as this will be removed in https://github.com/Shopify/kubernetes-deploy/pull/622.

Reply:
Good catch on the wrong variable name!